compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/AMDAIEDistributeCoresAndObjectFifos.cpp

```diff
@@ -480,9 +506,10 @@ LogicalResult insertLogicalObjectFifoAccess(ModuleOp moduleOp) {
     });
 
     WalkResult res = coreOp->walk([&](Operation *op) {
-      if (auto linalgOp = dyn_cast<linalg::LinalgOp>(op)) {
-        for (auto &&[idx, operand] :
-             llvm::enumerate(linalgOp->getOpOperands())) {
+      DenseMap<Value, AMDAIE::LogicalObjectFifoAccessOp>
+          newMemrefToLogicalObjectFifoAccess;
```

Comment:
Could you add a comment explaining what this new map is used for? Also, why are the memref's considered `new`? They're just the memref operands, no?

Reply:
Done.
Yes, they're memref operands - the `new` was added when I started working on the fix and because a "similar" map existed, I just added that as a prefix. :-P

I've updated the name now.